app/src/@libs/evm-wallet/hooks/useSwitchNetwork.ts

```diff
@@ -19,21 +19,20 @@ export const evmChainId = (chain: Chain) => {
 export const useSwitchNetwork = () => {
   const { updateTarget } = useDeploymentTarget();
 
-  const {
-    actions: { activate },
-    connectType,
-  } = useEvmWallet();
+  const { actions, connectType } = useEvmWallet();
+
+  const activateEvmWallet = actions?.activate;
```

Comment:
hmm, wondering now whether we actually need two methods to make this easier

activate(connectType)
switchChain(chainId)

?

Reply:
I thought `activate` also switches the chain. Isn't it?